library/src/morpheus-master/sass/modules/joinpage/_base.scss

```diff
@@ -0,0 +1,45 @@
+.#{$namespace}joinpage {
+    position: fixed;
+    width: 100%;
+    min-height: 14em;
+    background: #eee;
```

Comment:
is this not set via property on xlogin page?

Reply:
What are you referring to that is "set via property"?

Reply:
You are declaring a background color here instead of using a SASS property.

Reply:
Yes, looks like it should use $body-background-color instead, @bjones86.